config/dockerfiles/tools/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o jwt cmd/tools/jwt/jwt_cmd.go
 
 FROM golang:1.16 as downloader
 RUN go install github.com/linuxsuren/http-downloader@v0.0.49
-RUN http-downloader install kubesphere-sigs/ks@v0.0.58
+RUN http-downloader install kubesphere-sigs/ks@v0.0.59
```

Comment:
the PR https://github.com/kubesphere-sigs/ks/pull/245 has not been merged. so why bump it to v0.0.59 instead of v0.0.60

Reply:
Got it.